exercises/Python_Principles_2.ipynb

```diff
@@ -74,12 +68,13 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Naming variables can become very difficult in large programs, which may sound odd at first glance. When naming variables the choice of the name should be exact and concisely describe the data it contains. Using variables like `x` has no meaning when you need to look back at the code because it's easy to forget the meaning behind the variable.\n",
+    "Naming variables can become very difficult in large programs, which may sound odd at first glance. When naming variables the choice of the name should be exact and concisely describe the data it contains. A variable name like `x` has no meaning when you need to look back at the code because it's easy to forget what the variable contains.\n",
     "\n",
-    "Python does have some restrictions on how we name variables which are listed below. We've already encountered number 5 on the list already.\n",
```

Comment:
We could mention camel case and snake case here, and when/why you might use one or the other

Reply:
Yep I think I had this in a rough draft also, will add this today.